File: sdk/go-guest/examples/callee.go
package export_tests_local_api

import wit "go.bytecodealliance.org/pkg/wit/types"

func Answer() uint32 { return 42 }
func Fail() wit.Result[uint32,string] { return wit.Err[uint32,string]("declared application failure") }
func Spin() uint32 { for {} }

File: sdk/dotnet-guest/examples/callee.cs
namespace ServiceWorld.wit.Exports.tests.local;
public class ApiExportsImpl : IApiExports
{
    public static uint Answer() => 42;
    public static Result<uint, string> Fail() => Result<uint, string>.Err("declared application failure");
    public static uint Spin() { while (true) {} }
}
